x-pack/platform/test/cases_api_integration/common/plugins/cases/server/files/index.ts

```diff
@@ -16,7 +16,7 @@ const buildFileKind = (): FileKind => {
   return {
     id: CASES_TEST_FIXTURE_FILE_KIND_ID,
     http: fileKindHttpTags(),
-    allowedMimeTypes: ['image/png'],
+    allowedMimeTypes: ['image/png', 'text/plain'],
```

Comment:
Some files were uploading `text/plain` files when that mimeType wasn't originally allowed by this file kind:
```
./x-pack/platform/test/cases_api_integration/security_and_spaces/tests/common/internal/bulk_delete_file_attachments.ts
./x-pack/platform/test/cases_api_integration/security_and_spaces/tests/trial/internal/bulk_delete_file_attachments.ts
```